examples/spring-boot-minimal-kotlin/src/main/resources/config/application.yml

```diff
@@ -5,6 +5,6 @@ spring.main.web-application-type: none
 # Additional server ports specified with Armeria configuration.
 armeria:
   ports:
-  - port: 8080
+  - port: 0 # Specify the port you want.
```

Comment:
Is there any way to specify a different `application.yml` than the main one in test cases? I wish we can continue using port 8080 in the example.

Reply:
Fixed. 😄